### PR TITLE
CMO-222 track number of times profile clicked by user

### PR DESCRIPTION
**Pull request by pokharelmanish**
#### Which User story does this relate to (link)?
-
https://osi-cwds.atlassian.net/browse/CMO-222
#### Brief feature description
-logs number of times profile clicked by user. 

#### Do any commands needs to be run in order to run the project with this PR?
-no

### CODE REVIEW

**Comment by pokharelmanish** [comment]
Jenkins test this please

File: test/javascript/search/searchGrid_tests.js [hunk 1]
```diff
@@ -99,6 +112,13 @@ describe('Render Search results to Grid', () => {
   it('Verify Approval / Licensing worker is rendered', () => {
     expect(searchGridRender.find('GridInnerLayout[title="Approval / Licensing Worker"]').props().value).toBe('Wayne Fehlberg')
   })
+  it('WIP test onClick', () => {
```
Comment (pokharelmanish) [inline test/javascript/search/searchGrid_tests.js]: @TylerOSI Check this.

**Comment by pokharelmanish** [comment]
Jenkins test this please

File: app/javascript/search/searchGrid.js [hunk 2]
```diff
@@ -6,23 +6,26 @@ import GridOuterLayout from './common/gridOuterLayout.js'
...
+  userData
 }) => (
   <div className='grid_view col-xs-12 col-sm-12 col-md-12 col-lg-12'>
     {(
       searchResults.map((result, index) => {
+        const handleLogEvent = () => logEvent('profileClicked', {user: userData.user})
         return (<div key={index} className='grid_view_inner col-xs-12 col-sm-12 col-md-12 col-lg-12'>
           <div className='col-xs-12 col-sm-1 col-md-1 col-lg-1'>
-            <Link to={urlPrefixHelper('/facilities/' + result.id)}><div className='home-icon' /></Link>
+            <Link to={urlPrefixHelper('/facilities/' + result.id)} onClick={handleLogEvent}><div className='home-icon' /></Link>
```
Comment (TylerOSI) [inline app/javascript/search/searchGrid.js]: you can just make this an anonymous function

File: app/javascript/search/searchGrid.js [hunk 3]
```diff
@@ -6,23 +6,26 @@ import GridOuterLayout from './common/gridOuterLayout.js'
...
           </div>
           <div className='col-xs-12 col-sm-11 col-md-11 col-lg-11'>
             <GridOuterLayout>
               <div>
                 <p className='block_label'>Facility Name </p>
                 <p className='block_text'>
-                  <Link to={urlPrefixHelper('/facilities/' + result.id)}>{checkNullOrEmptyValue(result.name)}</Link>
+                  <Link to={urlPrefixHelper('/facilities/' + result.id)} onClick={handleLogEvent}>{checkNullOrEmptyValue(result.name)}</Link>
```
Comment (TylerOSI) [inline app/javascript/search/searchGrid.js]: you can also make this an anonymous function

**Pull request merged**